bootstrap.sh

```diff
@@ -40,14 +48,14 @@ echo ${2:-origin/master} > $PROVISIONING_FOLDER/ops_target_revision
 # * install librarian-puppet in Puppet internal ruby to download Puppet modules
 # * download a bootstrap Puppetfile
 # * download specified modules
-puppet apply $BOOTSTRAP_MANIFESTS_DESTINATION_FOLDER/bootstrap.pp --verbose --modulepath=modules
+run_puppet "$BOOTSTRAP_MANIFESTS_DESTINATION_FOLDER/bootstrap.pp --modulepath=modules"
```

Comment:
Why quote all parameters? This is probably a side effect of using `$*` rather than `"$@"` to expand parameters in the function.